src/scss/components/_dropdown.scss

```
@@ -5,53 +5,92 @@ $dropdown-item-padding: spaceUnits(2) spaceUnits(2);
+$icon-chevron-down-white: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" style="fill: rgba(255, 255, 255, 1);transform: ;msFilter:;"><path d="M16.293 9.293 12 13.586 7.707 9.293l-1.414 1.414L12 16.414l5.707-5.707z"></path></svg>');
+$icon-chevron-down-hover: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" style="fill: rgba(0, 123, 199, 1);transform: ;msFilter:;"><path d="M16.293 9.293 12 13.586 7.707 9.293l-1.414 1.414L12 16.414l5.707-5.707z"></path></svg>');
 $icon-world: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" style="fill: rgba(56, 72, 92, 1);transform: ;msFilter:;"><path d="M12 2C6.486 2 2 6.486 2 12s4.486 10 10 10 10-4.486 10-10S17.514 2 12 2zM4 12c0-.899.156-1.762.431-2.569L6 11l2 2v2l2 2 1 1v1.931C7.061 19.436 4 16.072 4 12zm14.33 4.873C17.677 16.347 16.687 16 16 16v-1a2 2 0 0 0-2-2h-4v-3a2 2 0 0 0 2-2V7h1a2 2 0 0 0 2-2v-.411C17.928 5.778 20 8.65 20 12a7.947 7.947 0 0 1-1.67 4.873z"></path></svg>');
 
+$animation-toggle: transform 0.2s ease-in-out;
+
 /* === Bootstrap Import === */
 @import '~bootstrap/scss/dropdown';
 
 /* === Custom styles === */
 
+.dropdown.show .btn.btn-dropdown.dropdown-toggle:after {
+  content: $icon-chevron-down;
+  transform: rotate(180deg);
+  transition: $animation-toggle;
+  height: 1.3em;
+}
+
 .btn .dropdown-chevron {
   padding: 2px;
 
   &:hover {
     background-color: transparent;
-    border-radius: 0.5rem;
+    border-radius: spaceUnits(2);
   }
 }
 
 .btn.btn-dropdown {
   padding: $btn-padding-y;
   font-weight: $body-font-weight;
+  border: 2px solid transparent;
+
+  &.btn-dropdown-border {
+    box-shadow: 0 0 0 2px #e6ebf0;
+    border-radius: spaceUnits(2);
+  }
 
   &:hover {
-    background-color: $grisulado-10;
-    border-radius: 0.5rem;
+    border-radius: spaceUnits(2);
+    box-shadow: 0 0 0 2px $blue;
+    text-decoration: underline;
+    color: $blue;
+
+    &.dropdown-toggle:after {
+      content: $icon-chevron-down-hover !important;
+    }
   }
 
   &:focus {
-    box-shadow: 0 0 0 0.2rem #00568b;
+    box-shadow: 0 0 0 0.2rem #005387;
     background: #fff;
     text-decoration: none;
-    border-radius: 0.5rem;
+    border-radius: spaceUnits(2);
   }
 
-  &.active {
-    color: $blue;
+  &.active,
+  &:active {
+    color: $white;
+    background: $blue;
+    border-radius: spaceUnits(2);
+    text-decoration: none;
+    box-shadow: none;
+
+    &.dropdown-toggle:after {
+      content: $icon-chevron-down-white !important;
+    }
   }
 
-  &.btn-dropdown-sm {
+  &.btn-dropdown {
+    min-height: 32px;
     padding: $btn-padding-y / 2 $btn-padding-y;
   }
 
+  &.btn-dropdown-sm {
```

No es necesario separar por clases.
Podes agregar dentro del mismo `.btn-dropdown` las clases de `lg` y `sm` de la siguiente manera:
```suggestion
 &.btn-dropdown {

    &-sm {
    }

    &-lg {
    }

  }
```